web/templates/shared/flashes.html.eex

```diff
@@ -0,0 +1,9 @@
+<%= for flash_key <- [:info, :error] do %>
+  <%= if flash = get_flash(@conn, flash_key) do %>
+    <div class="flash flash-<%= flash_key %>">
+      <div class="container">
+        <p data-role="flash"><%= raw flash %></p>
```

Comment:
Oh that makes a lot of sense. As long as we’re careful that nothing comes from the db (other than an id or something innocuous like that), then I think this is great

> On Apr 22, 2016, at 3:38 PM, Jason Draper <notifications@github.com> wrote:
> 
> In web/templates/shared/flashes.html.eex <https://github.com/drapergeek/community/pull/24#discussion_r60792379>:
> 
> > @@ -0,0 +1,9 @@
> > +<%= for flash_key <- [:info, :error] do %>
> > +  <%= if flash = get_flash(@conn, flash_key) do %>
> > +    <div class="flash flash-<%= flash_key %>">
> > +      <div class="container">
> > +        <p data-role="flash"><%= raw flash %></p>
> This came from another project. The reason we used the raw is that we often ended up wanting to show a link in the flash.
> 
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub <https://github.com/drapergeek/community/pull/24/files/5238dc5f2e286ee8cbd51e367bed6ef36dd7e645#r60792379>